db/versions/0091-migration.sql

```diff
@@ -0,0 +1,166 @@
+begin
+  -- migration is only possible when all tasks were migrated to new column structure
+  -- following query is commented out for documentation purposes, to prevent failed migration on prod deployment
```

Comment:
data is already ready on prod, so should be safe

Reply:
oh, indeed!
Then I will live it not commented, to give fail corrupting pending queues for deployments that will try to do run few upgrades at once. 
I'll also live a comment in changelog to say that this depends on successful migration from the previous release 👍